CreatubblesAPIClient/Sources/DAO/GalleryDAO.swift

```diff
@@ -105,12 +105,18 @@ class GalleryDAO: NSObject, APIClientDAO {
         return requestSender.send(request, withResponseHandler: handler)
     }
 
+    func getChallengeGalleries(_ pagingData: PagingData?, completion: GalleriesClosure?) -> RequestHandler {
+        let request = ChallengesGalleriesRequest(page: pagingData?.page, perPage: pagingData?.pageSize)
+        let handler = GalleriesResponseHandler(completion: completion)
```

Comment:
Yes - challenges are just regular galleries with one extra field and relationships